mslib/msui/mscolab.py

```diff
@@ -1410,22 +1422,78 @@ def add_operations_to_ui(self):
                 logging.debug("adding operations to ui")
                 operations = sorted(self.operations, key=lambda k: k["path"].lower())
                 self.ui.listOperationsMSC.clear()
+                self.ui.listInactiveOperationsMSC.clear()
                 selectedOperation = None
                 for operation in operations:
                     operation_desc = f'{operation["path"]} - {operation["access_level"]}'
-                    widgetItem = QtWidgets.QListWidgetItem(operation_desc, parent=self.ui.listOperationsMSC)
+                    widgetItem = QtWidgets.QListWidgetItem(operation_desc)
                     widgetItem.active_operation_desc = operation["description"]
                     widgetItem.op_id = operation["op_id"]
                     widgetItem.access_level = operation["access_level"]
                     widgetItem.operation_path = operation["path"]
                     widgetItem.operation_category = operation["category"]
+                    widgetItem.state = operation["state"]
                     if widgetItem.op_id == self.active_op_id:
                         selectedOperation = widgetItem
-                    self.ui.listOperationsMSC.addItem(widgetItem)
+                    if widgetItem.state == "active":
+                        self.ui.listOperationsMSC.addItem(widgetItem)
+                    else:
+                        self.ui.listInactiveOperationsMSC.addItem(widgetItem)
                 if selectedOperation is not None:
                     self.ui.listOperationsMSC.setCurrentItem(selectedOperation)
                     self.ui.listOperationsMSC.itemActivated.emit(selectedOperation)
                 self.ui.listOperationsMSC.itemActivated.connect(self.set_active_op_id)
+                self.ui.listInactiveOperationsMSC.itemActivated.connect(self.select_inactive_op)
+            else:
+                show_popup(self.ui, "Error", "Session expired, new login required")
+                self.logout()
+        else:
+            show_popup(self.ui, "Error", "Your Connection is expired. New Login required!")
+            self.logout()
+
+    def unselect_operation(self, op_id, item):
+        font = QtGui.QFont()
+        for i in range(self.ui.listInactiveOperationsMSC.count()):
+            self.ui.listInactiveOperationsMSC.item(i).setFont(font)
+        font.setBold(False)
+        item.setFont(font)
+
+    def select_inactive_op(self, item):
```

Comment:
select_inactive_op /  select_inactive_operation for making it consistant. 
 
 (We could later refactor all names to use the shortcut _op)